packages/bridge-ui/src/components/TokenDropdown/TokenDropdown.svelte

```diff
@@ -221,7 +221,7 @@
         <span class="title-subsection-bold text-base text-secondary-content">{$t('token_dropdown.label')}</span>
       {:else if value}
         <div class="flex f-space-between space-x-2 items-center text-secondary-content">
-          <!-- Only match icons to configurd tokens -->
+          <!-- Only match icons to configured tokens -->
```

Comment:
please dont include this change, or include it as part of a separate PR. PR should only modify a single package ideally.